libqtile/scripts/run_cmd.py

```diff
@@ -86,6 +90,11 @@ def add_subcommand(subparsers, parents):
         help='Set the window group.')
     parser.add_argument(
         'cmd',
+        help='Command to execute.'),
```

Comment:
It doesn't work to just add `required=True` here?

Reply:
No. That was my first try too. If you do that you get `TypeError: 'required' is an invalid argument for positionals`.

I also tries using `nargs='+'` but that didn't work as it tries to consume any additional arguments for itself, rather than passing them to `cmd`. For example, running `qtile run-cmd xeyes -fg "#ff0000"` will give the following when parsing the command:
``` python
Namespace(log_level='WARNING', socket=None, intrusive=False, float=True, dont_break=False, group='#ff0000', cmd=['xeyes'], func=<function run_cmd at 0x7f478331aca0>)
```
Note that the `fg` argument which should set xeyes's foreground colour is now the `group` argument.

The solution I've posted should avoid those issues.